### DISCUSSION TITLE
Login verification code bug (or How to file a bug report for GitHub.com itself ...)

### DISCUSSION THREAD

**Discussion by ceever** [community/community#47574] [Other Feature Feedback, Questions, & Ideas] [answered]
### GitHub.com Verfication Code Bug

When login in through TorBrowser on GitHub I regularly get a verification code sent to my email, which I have to provide on the website before being able to continue. This is no issue.

However, when typing the six numbers of the verification code, Ajax automatically starts verifying the code after the sixth number. This turns into an issue when typing the wrong code, then the verification continues even when correcting the just typed verification code, and only when the Ajax verification has verified the code to be wrong, it is possible to re-verify the corrected code. It will take up to 10 seconds to verify the code, where nothing can be done, even though one already started correcting the verification code.

It would be more useful, if the Ajax verification would be stopped/dropped as soon as the websites notices that the user starts correcting the code.

---

Sooo, I am not quite sure where to put this issue, which is why I write it to this forum. Maybe someone can point me to the right place.

Many thanks 👍

**Comment by unknowVariable** [comment]
Hey Ceever ! 

It sounds like you are experiencing an issue with the login verification process on GitHub. While it is a security feature to ensure that only authorized users are accessing the account, it can be frustrating if the verification code is not recognized or if there is a delay in the verification process.

One way to potentially resolve this issue is to try using a different web browser or clearing the cache and cookies on your current browser. Sometimes, browser extensions or cached data can interfere with website functionality. Additionally, make sure that your email provider is not blocking the verification code email from GitHub.

If the issue persists, you can try contacting GitHub support for further assistance. They may be able to provide more information on the issue and offer solutions to resolve it.

If this answer helps you or other, do not hesitate to select it as an answer I would be happy

**Comment by ceever** [comment]
I am sorry, but did you even read what I have written?

I was not looking for a generic answer, I was proposing an improvement to the website for a potential bug.

**Reply by unknowVariable** [reply]
Thank you for your politeness, but for writing in the right place and contact GitHub's support team about an issue or suggestion, you can follow these steps:

1. Go to the GitHub support page (https://support.github.com/)
2. Scroll down to the "Contact us" section and click on the "Contact us" button
3. Select the appropriate category that best matches your issue from the dropdown list
4. Follow the instructions to provide a detailed description of the issue and any relevant information, including screenshots or error messages if applicable
5. Once you have submitted your support request, you should receive a confirmation email from GitHub with a ticket number. GitHub's support team will review your request and respond to you as soon as possible.

Alternatively, you can also reach out to GitHub's support team by sending an email to [support@github.com](mailto:support@github.com). Make sure to include a detailed description of your improvement and any relevant information in your email.

**Comment by ceever** [comment]
One harvests what one saws. 🤷

Thanks for the actual answer. I will do as proposed.

**Comment by unknowVariable** [comment]
I apologize for trying to help you when my English is not perfect and leads me to misunderstandings :)

I wish you the best mate !

**Comment by ceever** [comment, marked as answer]
Your description sounds nice, but there is no such section "Contact us" under https://support.github.com/ ... and yet you wonder why I react impolitely.

For everyone else (💬), the following might help you more in case you encounter a similar issue/question:
1. Go to https://support.github.com/request
2. Navigate to "Help with GitHub Pages" and afterwards select either of the first two options "Help with ..."
3. This will lead you to a Contact Form.
4. At the top of the form there is also a link to a Bug Form (!) to file a bug report.

Generally, a description on what to include can be found here: https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket (This also has the link to https://support.github.com/request ...)

Good luck everyone 🙏

**Reply by unknowVariable** [reply]
" but there is no such section "Contact us" under https://support.github.com/ ... and yet you wonder why I react impolitely " 
<img width="969" alt="Capture d’écran 2023-02-16 à 18 46 10" src="https://user-images.githubusercontent.com/64416579/219446869-d67e9a2b-3a52-4b65-86e5-9327c8dee7fb.png">


Maybe it's not big enough 😂
not knowing seeks does not justify rudeness ;)

**Reply by ceever** [reply]
![20230216 190339_rectangle_1995x1226](https://user-images.githubusercontent.com/40247591/219450305-df50ddf5-dffe-48e2-a5a6-8831720d94af.png)

**Reply by unknowVariable** [reply]
<img width="1229" alt="Capture d’écran 2023-02-16 à 19 30 51" src="https://user-images.githubusercontent.com/64416579/219455965-ad2e64c5-cafe-49a2-9ce3-26927bc61e90.png">
<img width="760" alt="Capture d’écran 2023-02-16 à 19 31 03" src="https://user-images.githubusercontent.com/64416579/219455982-f318efe5-62d8-4137-a72a-ef9a40fc5cd9.png">
